cypress/integration/data_import/view-all.spec.js

```diff
@@ -0,0 +1,91 @@
+import TopMenu from '../../support/fragments/topMenu';
+import {
+  Select,
+  Heading,
+  Button,
+  Page,
+  TextField,
+  MultiColumnList,
+  including,
+} from '../../../interactors';
+
+// util for visiting View all page
+const gotoViewAllScreen = () => cy.do([Button('View all').click()]);
+
+// util for searching with term
+const searchWithTerm = (term) => (
+  cy.do([
+    TextField('Search ').fillIn(term),
+    Button('Search').click(),
+  ]));
+
+// util for selecting option
+const selectOption = (option) => cy
+  .do([Select({ id: 'input-job-logs-search-qindex' }).choose(option)]);
+
+describe('ui-data-import: "View all" log screen', () => {
+  before('uploads a file', () => {
+    cy.login(
```

Comment:
I see login twice - in before and beforEach. I expect to see such login only into one place